kortex_driver/src/non-generated/driver/kortex_arm_simulation.cpp

```diff
@@ -812,12 +812,35 @@ kortex_driver::KortexError KortexArmSimulation::ExecuteReachJointAngles(const ko
 
     // Transform kortex structure to trajectory_msgs to fill endpoint structure
     trajectory_msgs::JointTrajectoryPoint endpoint;
+    std::vector<int> limited_joints; // joints limited in range
```

Comment:
`std::unordered_set<int>` will result in better performance because down there you can just do `limited_joints.count(i)` and the operation will be constant complexity instead of iterating through each element.
in here it's absolutely negligible but elsewhere you might gain performance

Reply:
done